src/Orckestra.Composer/Services/ScopeViewService.cs

```diff
@@ -48,6 +50,16 @@ public virtual async Task<CurrencyViewModel> GetScopeCurrencyAsync(GetScopeCurre
             return vm;
         }
 
+        public async Task<Scope> GetScopeAsync(string scopeId)
+        {
+            var p = new GetScopeParam
+            {
+                Scope = scopeId
+            };
```

Comment:
ViewServices classes are use to return ViewModels but not service models.
If you need somewhere the Orckestra.Overture.ServiceModel.Scope then use ScopeRepository directly